tests/integ/scala/test_dataframe_join_suite.py

```diff
@@ -161,6 +161,10 @@ def test_join_with_multiple_conditions(session):
     assert res == []
 
 
+@pytest.mark.skipif(
+    "config.getvalue('local_testing_mode')",
+    reason="TODO: match error behavior",
```

Comment:
jira for error experience: https://snowflakecomputing.atlassian.net/browse/SNOW-1235716